Autopilot/Inc/gps.hpp

```diff
@@ -55,7 +42,56 @@ class Gps
 		* or with old data (in case of old data, the dataIsNew flag of the result struct will be cleared).
 		* @param[out]		Data 		pointer to the result struct.
 		*/
-        virtual void GetResult(GpsData_t *Data) = 0;
+        virtual void GetResult(GpsData_t &Data) = 0;
 };
 
+class NEOM8 : public Gps 
+{
+	public:
+		NEOM8(const NEOM8*) = delete;
+		NEOM8* GetInstance();
+
+		/**
+         * Triggers interrupt for new GPS measurement - stores raw data in variables and returns right away
+         * */
+		void BeginMeasuring();
```

Comment:
looks like this sensor sends data over by itself, so we never actually need to call this. Cool. You can write it as void BeginMeasuring(){}; and tell the user to not bother calling it.

Reply:
Something needs to call get_gps_data() in order for the NMEA strings to be parsed. We may need this to start parsing the data in the buffers...

Reply:
How about you store the raw data, then parse it at the time GetResult is called.

Reply:
That's a great idea! Will do that!